chart/templates/_helpers.tpl

```diff
@@ -107,3 +107,12 @@ Usage:
 {{- printf "%d" (add $i 1) }}
 {{- end }}
 {{- end }}
+
+{{/*
+Adds the project name
+Usage:
+{{ include "label_prefix" . }}/release: {{ .Release.Name }}
+*/}}
+{{- define "label_prefix" -}}
+    {{ print "openebs.io" }}
```

Comment:
should this be coming from a values.yaml var?

Reply:
Values yaml parameters are user-configurable. This is essentially a part of the template and not intended to be altered. Altering this may interfere with upgrade.

Reply:
Makes sense, then let's read it from a file. Perhaps other things in the values should also come from a file.

Reply:
I don't think helm template functions allow filesystem access. Any idea how I could approach this one?

Reply:
Added a yaml with 'openebs.io' as one of the constants. Good one!

Reply:
Awesome :+1: